wheel/README.md

```diff
@@ -1 +1,12 @@
 The `chia_rs` wheel contains python bindings for code from the `chia` crate.
+
+To run the tests:
+```
+cd wheel
+python -m venv venv
+. ./venv/bin/activate
+python -m pip install -r requirements.txt
```

Comment:
Consider instead

1. `python -m pip install -e .` (for a "development install" which points to the python files in the repo rather than making a pristine own copy that can let you "try out" python changes without reinstalling) or 
2. `python -m pip install .` (to make a pristine own copy that ignore further repo changes).